README.md

```diff
@@ -33,6 +36,10 @@ Now you're ready to start the server! ✨
 npm start
 ```
 
+Once the server is started, you will have the browser opened at http://localhost:3000 (and then it'll be immediately redirected to https://sparklespaces.com/). To start surfing the app, you can use the following url http://localhost:3000/in/devaliashacksville.
+
```

Comment:
If we're keeping the `Frontend` title above (line 10), you should probably add a `Backend` title too around here (before/after the note about frontend not working without backend).